src/engraving/libmscore/ornament.cpp

```diff
@@ -116,7 +116,7 @@ PropertyValue Ornament::propertyDefault(Pid id) const
     case Pid::INTERVAL_BELOW:
         return OrnamentInterval(IntervalStep::SECOND, IntervalType::AUTO);
     case Pid::ORNAMENT_SHOW_ACCIDENTAL:
-        return OrnamentShowAccidental::DEFAULT;

```

Comment:
Isn't it weird if the enum value called `DEFAULT` is not the default? Maybe we should rename `OrnamentShowAccidental::DEFAULT` to `OrnamentShowAccidental::AUTO`?

Reply:
Thanks, renamed